ppb/engine.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, first_scene: Type, *,
         self.event_extensions = defaultdict(dict)
         self.running = False
         self.entered = False
+        self.last_idle_event = None
```

Comment:
This is poorly named.

Reply:
It started as "last_heartbeat". Would "last_idle_timestamp" be better?